### PR TITLE
Change Rules docs to show ECMAScript 6 support

### PR DESCRIPTION
**Pull request by jwineinger**
Update the Rules docs to state that ECMAScript 6 is supported, since webtask.io now supports it.

### CODE REVIEW

File: articles/rules/current/index.md [hunk 1]
```diff
@@ -349,7 +349,7 @@ Notice that the code sandbox in which Rules run on, can be recycled at any time.
 
 ## Available modules
 
-For security reasons, the Rules code runs in a JavaScript sandbox based on [webtask.io](https://webtask.io) where you can use the full power of the ECMAScript 5 language.
+For security reasons, the Rules code runs in a JavaScript sandbox based on [webtask.io](https://webtask.io) where you can use the full power of the ECMAScript 6 language.
```
Comment (tjanczuk) [inline articles/rules/current/index.md]: I would rephrase it completely to something like:

> For security reasons, your Rules code executes isolated from the code of other Auth0 tenants in a sandbox based on [Extend](https://goextend.io?utm_source=docs&utm_medium=page&utm_campaign=auth0-com&utm_content=docs-rules). Within the sandbox, you have access to the full power of Node.js with a large number of Node.js modules.

Comment (tjanczuk) [reply articles/rules/current/index.md]: Please note the full link going to `https://goextend.io?utm_source=docs&utm_medium=page&utm_campaign=auth0-com&utm_content=docs-rules`.

File: articles/rules/current/index.md [hunk 2]
```diff
@@ -349,7 +349,7 @@ Notice that the code sandbox in which Rules run on, can be recycled at any time.
...
 
 For a list of currently supported sandbox modules, see [Modules Supported by the Sandbox](https://tehsis.github.io/webtaskio-canirequire) and [Additional Modules Available in Rules](/appliance/modules).
```
Comment (tjanczuk) [inline articles/rules/current/index.md]: FYI The "canirequire" mechanism is going to undergo changes soon. For context see: https://auth0.slack.com/archives/C9170558S/p1529349217000479

Comment (mpaktiti) [reply articles/rules/current/index.md]: Thanks, I followed that issue so when the change is ready I can make any updates

**Comment by mpaktiti** [comment]
Thank you @jwineinger, I pushed some updates and will merge now!
